src/AppBundle/Form/ProcurationElectionsType.php

```diff
@@ -19,15 +19,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder
             ->add('electionPresidentialFirstRound', CheckboxType::class, [
                 'required' => false,
+                'data' => false,
```

Comment:
@tgalopin there was a change request to set the checkboxes to `unchecked` by default. per the documentation, it seemed to be how to handle it. let me know your thoughts!